modules/sdk-coin-eth/src/lib/utils.ts

```diff
@@ -579,7 +579,7 @@ type RecursiveBufferOrString = string | Buffer | RecursiveBufferOrString[];
 export function getRawDecoded(types: string[], serializedArgs: Buffer): RecursiveBufferOrString[] {
   function normalize(v: unknown, i: number): unknown {
     if (BN.isBN(v)) {
-      return v.toBuffer();
+      return v;
     } else if (typeof v === 'string' || Buffer.isBuffer(v)) {
```

Comment:
can be combined into the same if block

Reply:
Will end users have to change their code as a result of this line change?

Reply:
likely not, it's only change when we refactor and it's not working since then.

Reply:
Cool, wanted to make sure that we didn't need to label the commit as BREAKING CHANGE. :)